### ISSUE TITLE
no shadows?

### ISSUE THREAD

**Issue by antithing** [adynathos/AugmentedUnreality#26]
Hi, thanks for making this code available. I have loaded the example project, and printed the markers. I see the cube and candle overlaid in the game, but I do not see the shadows.  Do I need to enable them somehow? Thanks again.

**Comment by natarajnattu** [comment]
even i'm also not getting shadows on shadow plane! any suggestions for this question @adynathos 

i tried put static mesh under the shadow plane and still not appearing.
played with lightning still don't know whats the issue.

**Comment by adynathos** [comment]
What OS are you running this on? I think there was a problem with Android rendering which didn't show the shadows which were present on Windows.

I will look at it next week when I'm back from holidays. 
I recommend inspecting then shadow plane material settings - maybe there is something about shadows.

**Comment by natarajnattu** [comment]
Thanks for the reply.. 
enjoy the holidays!

**Comment by adynathos** [comment]
I think the reason was that the shadow-plane actor was hidden in the example level, could you try the new test project (1.2.05) or un-hiding the shadow-plane?

**Comment by natarajnattu** [comment]
Hi adynathos,	


when im building source im geting this error in the file:
AURVideoScreenBase.cpp

error
 Severity	Code	Description	Project	File	Line	Suppression State
Error (active)		class "UAURVideoScreenBase" has no member "SetGenerateOverlapEvents"	AugmentedUnrealityEx	\Source\AugmentedUnreality\AURVideoScreenBase.cpp	line 34
 
when i commented out this line 	
this->SetGenerateOverlapEvents(false);

the build is succeed and when i open the project in 4.19 it says the project map is built for newer versions

when i tried to generate files with 4.20 version, it saying.
Visual Studio 2015 requires the Universal CRT to be installed.

pls help

**Comment by adynathos** [comment]
> is 1.2.05 project for 4.20?

Yes, the change 1.2.04 -> 1.2.05 is making it compile with UE4.20.

Anyway you don't need to use the new version to try the shadows.
In 1.2.04's example project, the shadow plane actor is "hidden in game". Find the shadow plane actor in the map and uncheck "hidden in game".

> when i tried to generate files with 4.20 version, it saying.
Visual Studio 2015 requires the Universal CRT to be installed.

That suggests that you could install some "universal CRT" component for your VS, or update to VS2017.
